storage/aptosdb/src/db/include/aptosdb_writer.rs

```diff
@@ -197,6 +199,76 @@ impl DbWriter for AptosDB {
             Ok(())
         })
     }
+
+    /// Revert a commit.
+    fn revert_commit(&self, ledger_info_with_sigs: &LedgerInfoWithSignatures) -> Result<()> {
```

Comment:
This is cherry-picked from #28, not yet reviewed there. As I wrote this code, I'll leave it as an exercise for @0xmovses.